pymobiledevice3/lockdown.py

```diff
@@ -441,8 +442,13 @@ def pair(self, timeout: int = None) -> None:
         self.paired = True
 
     @reconnect_on_remote_close
-    def unpair(self) -> Mapping:
-        return self._request('Unpair', {'PairRecord': self.pair_record, 'ProtocolVersion': '2'}, verify_request=False)
+    def unpair(self, host_id: str = None) -> Mapping:
```

Comment:
- change to `-> None`
- don't return, simply request it

Reply:
done